src/VisualStudio/Core/Def/InheritanceMargin/MarginGlyph/MemberMenuItemViewModel.cs

```diff
@@ -35,26 +37,15 @@ internal class MemberMenuItemViewModel : MenuItemViewModel
         public MemberMenuItemViewModel(
             string displayContent,
             ImageMoniker imageMoniker,
-            string automationName,
-            ImmutableArray<MenuItemViewModel> targets) : base(displayContent, imageMoniker, automationName)
+            ImmutableArray<MenuItemViewModel> targets) : base(displayContent, imageMoniker)
```

Comment:
all code passed the displayContent along as teh automation name.  so all that redundancy was removed and the base type just does that sharing in its constructor.